public/pages/MonitorDetails/containers/MonitorDetails.js

```diff
@@ -174,14 +178,189 @@ export default class MonitorDetails extends Component {
     this.setState({ triggerToEdit: null });
   };
 
-  onEditTrigger = trigger => {
+  onEditTrigger = (trigger) => {
     this.setState({ triggerToEdit: trigger });
     this.props.history.push({
       ...this.props.location,
       search: `?action=${TRIGGER_ACTIONS.UPDATE_TRIGGER}`,
     });
   };
 
+  convertToADConfigs = async (monitor) => {
+    const uiMetadata = _.get(monitor, 'ui_metadata');
+    let adName = monitor.name + '-Detector';
+    let adTimeField = uiMetadata.search.timeField;
+    let adIndices = monitor.inputs[0].search.indices;
+    let adDetectorInterval = { period: { interval: 1, unit: 'MINUTES' } };
+    const {
+      frequency,
+      period: { interval, unit },
+      daily,
+      weekly,
+      monthly: { day },
+      cronExpression,
+      timezone,
+    } = _.get(uiMetadata, 'schedule', {});
+    const search = _.get(uiMetadata, 'search');
+    if (frequency === 'interval') {
+      adDetectorInterval = { period: { unit: unit, interval: interval } };
+    }
+    let windowDelay = await this.getLatestTimeStamp(adTimeField, adIndices);
+    let adWindowDelay = { period: { interval: 10, unit: 'MINUTES' } };
+    if (adWindowDelay) {
+      adWindowDelay = { period: { interval: windowDelay, unit: 'MINUTES' } };
+    }
+    let filterQuery = formikToWhereClause(search);
+    let adFilterQuery = '';
+    if (filterQuery) {
+      adFilterQuery = { bool: { filter: [filterQuery] } };
+    } else {
+      adFilterQuery = { match_all: { boost: 1.0 } };
+    }
+    let { aggregationType, fieldName } = search;
+    let adFeatures;
+    if (!aggregationType || !fieldName) {
+      adFeatures = {};
+    } else {
+      if (aggregationType == 'count') {
+        aggregationType = 'value_count';
+      }
```

Comment:
what is the reason for this? Is there a discrepancy between naming conventions for the count aggregation between alerting and AD, but the rest of the aggregations match up?